packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -1377,44 +1379,37 @@ export const components = (
           },
 
           ".Link-indicator": {
-            marginInlineStart: theme.spacing(2),
+            marginInlineStart: odysseyTokens.Spacing2,
           },
 
           ".Link-icon": {
-            marginInlineEnd: theme.spacing(2),
+            marginInlineEnd: odysseyTokens.Spacing2,
           },
           svg: {
-            fontSize: "1rem",
```

Comment:
How does this change look effectively?

Reply:
It's identical, since the `font-size` on the parent is `1rem`, but now the icon will scale if that changes